### PR TITLE
[PyTorch-nightly-test] Fix Wav2Vec2 LM & Phoneme tests

### PR DESCRIPTION
**Pull request by patrickvonplaten**
# What does this PR do?

Install packages for `pytorch-nightly` tests to not fail

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: .github/workflows/self-nightly-scheduled.yml [hunk 1]
```diff
@@ -33,9 +33,11 @@ jobs:
 
             - name: Install dependencies
               run: |
-                  apt -y update && apt install -y libsndfile1-dev git
+                  apt -y update && apt install -y libsndfile1-dev git espeak-ng
                   pip install --upgrade pip
+                  pip install https://github.com/kpu/kenlm/archive/master.zip
```
Comment (patrickvonplaten) [inline .github/workflows/self-nightly-scheduled.yml:38]:
```suggestion
```

**Comment by sgugger** [comment]
Hey @patrickvonplaten, could you write a description in the PR on why the change is needed?

File: .github/workflows/self-nightly-scheduled.yml [hunk 2]
```diff
@@ -152,10 +154,11 @@ jobs:
 
             - name: Install dependencies
               run: |
-                  apt -y update && apt install -y libaio-dev
+                  apt -y update && apt install -y libsndfile1-dev git espeak-ng
```
Comment (stas00) [inline .github/workflows/self-nightly-scheduled.yml:157]: why have you removed `libaio-dev`?

File: .github/workflows/self-nightly-scheduled.yml [hunk 3]
```diff
@@ -193,11 +196,12 @@ jobs:
...
```
Comment (stas00) [inline .github/workflows/self-nightly-scheduled.yml:199]: here as well

**Review by stas00 — CHANGES_REQUESTED** [review]
LGTM, but please restore the essential `libaio-dev` (needed for deepspeed infinity testing)

File: .github/workflows/self-nightly-scheduled.yml (hunk 2 above)
Comment (patrickvonplaten) [reply .github/workflows/self-nightly-scheduled.yml:157]: ups - sorry about that!

Comment (patrickvonplaten) [inline .github/workflows/self-nightly-scheduled.yml:157]:
```suggestion
                  apt -y update && apt install -y libaio-dev libsndfile1-dev git espeak-ng
```

File: .github/workflows/self-nightly-scheduled.yml (hunk 3 above)
Comment (patrickvonplaten) [inline .github/workflows/self-nightly-scheduled.yml:199]:
```suggestion
                  apt -y update && apt install -y libaio-dev libsndfile1-dev git espeak-ng
```

**Comment by patrickvonplaten** [comment]
Thanks for the careful review @stas00 !

**Review by stas00 — APPROVED** [review]
LGTM

**Pull request merged**